module/icp/algs/aes/aes_impl.c

```diff
@@ -405,6 +405,7 @@ aes_impl_set(const char *val)
 	return (err);
 }
 
+#ifndef __APPLE__
 #if defined(_KERNEL)
```

Comment:
Let make it clear this is still Linux specific, and not just `!__APPLE__`

```suggestion
#if defined(_KERNEL) && defined(__linux__)
```

Reply:
I thought FreeBSD used them as well? But yes, that would be cleaner.

Reply:
They do, but currently FreeBSD isn't building this file so it hasn't yet been updated to use `ZFS_MODULE_PARAM_CALL`.  The format of the callback handlers is different enough we needed to move that code to per-platform `_os.c` files.